modular_skyrat/master_files/code/modules/mob/living/human/species_types/flypeople.dm

```diff
@@ -0,0 +1,2 @@
+/datum/species/fly
+	allow_food_preferences = FALSE
```

Comment:
suggestion: you may want to addendum this to synth's aswell, or, more importantly, hemophages because of: https://github.com/Skyrat-SS13/Skyrat-tg/pull/18292 (hopefully it's merged first to avoid issues)

Reply:
Neither synths or hemophages have food item requirements, and hemophage hunger is based off their blood levels, no?

If this is the case, then this just becomes mostly flavor, though it's quite easily disabled if need be.

Reply:
> 

tbh there aren't many mood buffs synths have but still have all the debuffs (Surgery is always one you get cucked on)

Reply:
> Neither synths or hemophages have food item requirements, and hemophage hunger is based off their blood levels, no?
> 
> If this is the case, then this just becomes mostly flavor, though it's quite easily disabled if need be.

Hemophages have some food preferences now, since the third part of the Revamp PR series has been merged.